### PR TITLE
gigasecond: standardize starter implementation

### LINKED ISSUE #538: gigasecond: create and/or validate starter implementations
Ref.: https://github.com/exercism/xjava/blob/master/POLICIES.md#starter-implementations

This exercise is currently one of the first ten exercises in the track. Please review the exercise files and create and/or update the starter implementation according to the guidelines provided in the link above.

### PR DESCRIPTION
**Pull request by stkent**
Fixes #538.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/gigasecond/src/example/java/Gigasecond.java [hunk 1]
```diff
@@ -1,19 +1,20 @@
 import java.time.LocalDateTime;
 import java.time.LocalDate;
 
-public class Gigasecond {
+class Gigasecond {
 
-    private LocalDateTime birthDateTime;
+    private LocalDateTime initialDateTime;
```
Comment (FridaTveit) [inline exercises/gigasecond/src/example/java/Gigasecond.java]: I agree that calling it `initialDateTime` makes the class more general and might be nicer. But calling it `birthDateTime` lines up more with the description of the exercise and might make it easier for people solving it to understand what they need to do? I don't particularly mind though :)

Comment (stkent) [inline exercises/gigasecond/src/example/java/Gigasecond.java]: I feel like we've had a very similar discussion before, so apologies, but: I was referring to the [canonical description](https://github.com/exercism/x-common/blob/master/exercises/gigasecond/description.md) for this problem and it doesn't mention birthdates (which is why I made the adjustment I did). Please remind me where I can find the reference you're thinking of? Is the [blurb](https://github.com/exercism/x-common/blob/master/exercises/gigasecond/metadata.yml#L2) always combined with the `description.md` file? Thanks!

Comment (FridaTveit) [inline exercises/gigasecond/src/example/java/Gigasecond.java]: Yes, that was what I was thinking of :)

**Comment by stkent** [comment]
@FridaTveit updated!

**Pull request merged**